jenkins/run

```diff
@@ -59,12 +51,12 @@ fi
 podman run \
     --pull=always \
     --userns=keep-id \
-    --volume $(pwd):/src/pbench:z \
-    --volume ${HOME_VOLUME} \
+    --volume $(pwd):${HOME_DIR}/pbench:z \
+    --volume ${HOME_DIR} \
     ${GIT_BASE_VOLUME} \
-    -w /src/pbench \
-    --env-host \
-    --env HOME=${IHOME} \
+    -w ${HOME_DIR}/pbench \
+    --env HOME=${HOME_DIR} \
+    --env USER=${USER_NAME} \
```

Comment:
Since, for good reasons, we are no longer using `--env-host`, we need to define certain environment variables inside the container.  The definition of `HOME` is pretty obvious and straightforward, since I don't think that there are very many constraints on it, and it is dictated by the setup we've done with the mounted volumes.  However, I think that setting up `USER` is a little bit trickier.  In particular, I think we should be wary of setting it to an arbitrary value.

Under normal circumstances, I believe that `USER` is set to a value which comes from `/etc/passwd` or its distributed equivalent.  And, I think the value matches what is returned by `id -nu`.  However, with the proposed change, inside the container, the value of `USER` _won't_ match the `id` value.  I suspect that the result of that will be Bad, e.g.:
```
# echo $USER
developer
# id $USER
id: 'developer': no such user
# whoami
root
```
I'm concerned that this is going to lead to some sort of identity crisis.

Reply:
Not sure if I share the concerns, but let's see if a simple change does the trick.